mlir/lib/Conversion/GPUToMIGraphX/CMakeLists.txt

```diff
@@ -19,5 +19,4 @@ target_link_libraries(MLIRGPUToMIGraphX
   MLIRMIGraphX
   MLIRGPUToGPURuntimeTransforms
   MLIRGPUToROCDLTransforms
-  LLVMROCmBackendUtils
```

Comment:
Let's call it MLIRRocmBackendUtils?

Reply:
@jerryyin What do you think about the naming?

Reply:
Just as FYI, we talked a second ago and sounds like a better suited name is: `IsaNameSplitter`. This is as close as it can get in what this component does.